etwpipeline/akamai/client.py

```diff
@@ -244,3 +252,14 @@ def describe_property_by_id(self, property_id: str) -> PropertyDescription:
         return PropertyDescription(
             id=property_id, name=property_name, origins=list(origins), edge_hosts=list(edge_hosts)
         )
+
+    ## GTM functions
+    def list_gtm_domains(self):
+        gtm_domains_path = "/config-gtm/v1/domains"
+        return self._get_api_from_relative_path(gtm_domains_path)['items']
+    
+    def get_gtm_domain(self, domain_name: str):
+        gtm_domain_path = (
+            f"/config-gtm/v1/domains/{domain_name}"
+        )
+        return self._get_api_from_relative_path(gtm_domain_path, headers = {'accept': 'application/vnd.config-gtm.v1.5+json'})
```

Comment:
Lets go over this together if you don't mind.  

I believe it would be nice to build a result set object somewhere here (not sure yet):
- CNAME: subdomain_name.gtm_domain_name (concat name+gtm_domain_name)
    - CNAME -> IP (ie. ww1.subdomain_name.gtm_domain_name - x.x.x.x)
    - CNAME -> IP (ie. ww2.subdomain_name.gtm_domain_name - x.x.x.2)
    - CNAME -> IP (ie. ww3.subdomain_name.gtm_domain_name - x.x.x.3)